godtools/App/Features/Menu/Domain/UseCases/GetUserActivityBadgeUseCase/UserActivityBadgeDomainModel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UserActivityBadgeDomainModel.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 1/17/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import GodToolsToolParser
```

Comment:
With the ```Badge.BadgeType``` removed we no longer need the ```GodToolsToolParser``` dependency here which does make this more clean since the domain model doesn't know about the data layer.
We haven't been fully implementing clean in our UseCases since those do reference our concrete data layer classes, but maybe in the future that is something we can start to integrate as we create a more distinct data layer.